wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -40,9 +28,9 @@ const initDraftailEditor = (fieldName, options = {}) => {
   if (options.entityTypes) {
     // eslint-disable-next-line no-param-reassign
     options.entityTypes = options.entityTypes.map(entity => Object.assign(entity, {
-      control: controls[entity.type],
+      control: sources[entity.source],
```

Comment:
Please add a comment mentioning that this should be renamed once https://github.com/springload/draftail/issues/49 is done.